frontend/src/components/getStarted/FAQ.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import QandA from './QandA.jsx';
+import Collapse from 'rc-collapse';
+
+
+class FAQ extends React.Component {
+  render() {
+    return (
+      <div className="FAQ">
+        <div className="container">
+          <h1>FAQ</h1>
+          <QandA 
+          question={"Do I need to be from UCL to develop?"}
+          answer={["Currently to develop applications you must be affiliated with UCL as authentication is done via UCL login. Without an appropriate login you will be unable to develop applications using UCL API."]}
+          id={"1"} />
+
+          <QandA 
+          question={"Do I need to use a particlar language?"}
+          answer={["UCL API is a RESTful API meaning that you are not restricted to using any specific language.  Our ", <a href='/docs'>documentation</a>, " currently includes instructions on how to get up and running with Javascript, Python and Shell. However there is no restraints on the language you could use."]}

```

Comment:
```suggestion
          answer={["UCL API is a RESTful API meaning that you are not restricted to using any specific language. Our ", <a href='/docs'>documentation</a>, " currently includes instructions on how to get up and running with Javascript, Python and the Unix shell using cURL. However, you may use any other programming language so long as it can make web requests."]}
```